packages/@aws-cdk/aws-eks/test/test.cluster.ts

```diff
@@ -281,7 +285,7 @@ export = {
     expect(stack).to(haveResource('AWS::AutoScaling::AutoScalingGroup', {
       Tags: [
         {
-          Key: { 'Fn::Join': ['', ['kubernetes.io/cluster/', { Ref: 'ClusterEB0386A7' }]] },
+          Key: { 'Fn::Join': ['', ['kubernetes.io/cluster/', { Ref: 'Cluster9EE0221C' }]] },
```

Comment:
Why did this change?

Reply:
Because I removed the `kubectlEnabled: false` and so the actual CFN resource is now the custom resource and not the original resource.

Reply:
👍